test/CMakeLists.txt

```diff
@@ -1,14 +1,12 @@
-add_executable(test_plist test_plist.c)
-add_executable(test_pstack test_pstack.c)
-add_executable(test_pqueue test_pqueue.c)
-add_executable(test_pdict test_pdict.c)
+include_directories(include)
+add_executable(test_pexcept_hooks ${CMAKE_SOURCE_DIR}/src/pexcept.c test_pexcept_hooks.c)
```

Comment:
It'd be nice to have a comment explaining why this particular test suite needs special treatment.